recipes/unleash-client-cpp/all/conanfile.py

```diff
@@ -0,0 +1,101 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class UnleashConan(ConanFile):
+    name = "unleash-client-cpp"
+    homepage = "https://github.com/aruizs/unleash-client-cpp/"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Unleash Client SDK for C++ projects."
+    topics = ("unleash", "feature", "flag", "toggle")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False]}
+    default_options = {"shared": False}
+    generators = "cmake", "cmake_find_package"
+    exports_sources = "CMakeLists.txt", "patches/**"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+    requires = (
+        "cpr/1.7.2",
+        "nlohmann_json/3.10.5",
+    )
+
+    build_requires = "cmake/3.22.0"
```

Comment:
Why is this required 🤔?

Reply:
I chose that cmake version to match the opensll version in the rest of dependencies. Another option would be to add the openssl explicitily in the recipe with an specific version.

Reply:
Anyway, I've updated the unleash-client-cpp library to reduce the minimum cmake required version.